templates/cases/case_read.haml

```diff
@@ -163,7 +163,10 @@
             .container-fluid
               .row
                 .contact-field-label.col-sm-6
-                  - trans "Name"
+                  - if contact_display_format == 'urn'
+                    - trans "URN"
```

Comment:
I don't think most users know what a URN is, so I think here we should list urns with their schemes. As elsewhere, "tel" can become "Phone" etc. If SITE_ANON_CONTACTS is true, then don't list any URNs.

```
Name     Bob                
Phone    +250353252111 
Email    bob@example.com     
```